addons-l10n/en/play-later.json

```diff
@@ -0,0 +1,6 @@
+{
+    "play-later/play-later-title": "Play Later",
+    "play-later/add": "Play Later",
```

Comment:
I separated them because I thought that they might sometimes translate differently, since one refers to the action of adding a project to be played later, while the other refers to the section title.